### ISSUE TITLE
Doesn't work with Homey 3.0 and appstore as of November2019

### ISSUE THREAD

**Issue by giggo** [squaretronics/com.squaretronics.newhomeyapps#4]
Would be great if the app could be updated to work with the new appstore. Thanks

**Comment by fkey3** [comment]
Agreed, especially since athom removed the notice of app updates. So now you need to go through every app one by one to see if there is an update...
